pkgs/development/beam-modules/lib.nix

```diff
@@ -12,6 +12,15 @@ rec {
 
   callPackage = callPackageWith pkgs;
 
+  /* Erlang/OTP-specific version retrieval, returns 19 for OTP R19 */
+  getVersion = x:
+   let
+     parse = drv: (builtins.parseDrvName drv).version;
+   in builtins.replaceStrings ["B" "-"] ["." "."] (
+      if stdenv.lib.isString x
```

Comment:
doesn't really matter, but `isString` is a builtin so you don't need stdenv.lib.